scripts/extract-markdown-fencedcodeblocks.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# Extracts fenced code blocks from the given markdown supplied on stdin, emitting them to stdout.
+# If the fenced code block has an attribute `adjunct` its contents are emitted too, before the codeblock to which it is
```

Comment:
The syntax I chose was inspired by https://python-markdown.github.io/extensions/fenced_code_blocks/

GitHub's Markdown [specification](https://github.github.com/gfm/#info-string) suggests that only the first word of the "info string" is significant to GitHub's implementation.   It seems to ignore the extra content without error.